### PR TITLE
Adding Sails.js Page

### PR DESCRIPTION
**Pull request by steventhanna**
Adding a page for [Sails.js](http://sailsjs.org/), one of the largest JavaScript frameworks built on top of Node.js

### CODE REVIEW

File: pages/common/sails.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+`sails lift`
+
+- Create new Sails project
+
+`sails new {{projectName}}`
+
+- Display the version
+
+`sails -v` or `sails --version`
```
Comment (igorshubovych) [inline pages/common/sails.md]: This is trivial. I would probably avoid writing this example. Unless it is java command, which has `-version` (single dash).

File: pages/common/sails.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Generate Sails Specific Files
+
+`sails generate {{type}} {{name}}`
```
Comment (igorshubovych) [inline pages/common/sails.md]: This is where the magic is.

Maybe it makes sense:
```md
- Explanation

`sails generate controller {{controller_name}}`

- Explanation

`sails generate api {{resource}}`
```

**Comment by igorshubovych** [comment]
LGTM

**Comment by felixonmars** [comment]
Please squash the commits, thanks!

File: pages/common/sails.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+sails
```
Comment (rubenvereecken) [inline pages/common/sails.md]: Almost missed this one. Please add a header tag, as in `# sails`. I'll be happy to merge then.

**Comment by igorshubovych** [comment]
Thank you for contribution!

**Pull request merged**